bridges/centralized-ethereum/src/actors/dr_reporter.rs

```diff
@@ -682,15 +681,11 @@ mod tests {
                 .parse()
                 .unwrap(),
             topics: vec![
-                "0x00e9413c6321ec446a267b7ebf5bb108663f2ef58b35c4f6e18905ac8f205cb2"
+                "0x4df64445edc775fba59db44b8001852fb1b777eea88fd54f04572dd114e3ff7f"
                     .parse()
                     .unwrap(),
             ],
-            data: web3::types::Bytes(vec![
-                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-                0, 0, 248, 117, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16, 232, 36, 130, 44, 106, 92,
-                40, 222, 53, 104, 223, 153, 96, 77, 104, 233, 253, 156, 140,
-            ]),
+            data: web3::types::Bytes(hex::decode("0000000000000000000000000000000000000000000000000000000000001b58000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000146e6f7420696e20506f7374656420737461747573000000000000000000000000").unwrap()),
```

Comment:
Well done. I hate how much we rely on byte arrays all across the code base and in our APIs. Hex is the way to go.